sktime/forecasting/timesfm_forecaster.py

```diff
@@ -370,6 +393,12 @@ def get_test_params(cls, parameter_set="default"):
             `create_test_instance` uses the first (or only) dictionary in `params`
         """
         test_params = [
+            {
+                "context_len": None,
+                "horizon_len": None,
+                "freq": 0,
+                "verbose": False,
+            },
```

Comment:
```suggestion
            {
                "context_len": None,
                "horizon_len": 128,
                "freq": 0,
                "verbose": False,
            },
```
this would raise exception otherwise for test suits that don't provide `fh` during `fit`